README.md

```diff
@@ -96,10 +96,31 @@ Ensure the following elements are present in your `manifest.yml`. The `run_migra
 To seed fund & round data to db
 
 ```
-docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r2
-docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r3w1
-docker exec -ti <fund_store_container_id> python -m scripts.load_ns_r2
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_cof_r2
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_cof_r3w1
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_cof_r3w2
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_ns_r2
 ```
+
+To find all load modules (from root of project):
+
+```bash
+find ./scripts -type f -name "load_*.py" | xargs -I {} basename {} | sed 's/\.py$//'
+```
+
+To dynamically load all:
+
+```bash
+find ./scripts -type f -name "load_*.py" -exec sh -c 'docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.$(basename {} .py)' \;
```

Comment:
so just one cmd to load all the fund/rounds config in db ...?.

Reply:
Yeah.  I think for now this is a little redundant since we only have 4 rounds, but say 5 months from now we may have 40-50, just an easier command to load them all at once rather than copy pasting each load script.

Although if we end up having that many maybe developers will only load relevant ones for development...